src/components/AuthDialog/login/index.tsx

```diff
@@ -65,11 +65,15 @@ const Login: React.FC<AuthDialogProps> = ({
                 </div>
                 <div className="grid w-full max-w-sm items-center">
                   <Input
+                    {...loginMethods.register('password')}
                     type="password"
                     id="password"
                     placeholder="Password"
                     className="bg-purple-200 hover:bg-purple-300"
                   />
+                  <p className="text-sm text-red-500">
+                    {loginMethods.formState.errors.password?.message || ''}
+                  </p>
```

Comment:
the entire p element should be rendered depending on the message state and use a span instead
should be like this:
`{loginMethods.formState.errors.password && <span>{loginMethods.formState.errors.password?.message}</span>}`